### ISSUE TITLE
problem with nextjs in dev mode

### ISSUE THREAD

**Issue by hannanstd** [teodosii/react-notifications-component#28]
hi
when i use your awesome package in nextjs (npm run dev) and i open the site with ssr. i get below error and page can not be loaded.

`ReferenceError: window is not defined
    at /var/www/html/rebion/react/node_modules/react-notifications-component/dist/js/react-notifications.dev.js:1355:2
    at /var/www/html/rebion/react/node_modules/react-notifications-component/dist/js/react-notifications.dev.js:1344:46
    at module.exports../node_modules/style-loader/lib/addStyles.js.module.exports (/var/www/html/rebion/react/node_modules/react-notifications-component/dist/js/react-notifications.dev.js:1412:88)
    at Object../src/scss/notification.scss (/var/www/html/rebion/react/node_modules/react-notifications-component/dist/js/react-notifications.dev.js:2853:137)
    at __webpack_require__ (/var/www/html/rebion/react/node_modules/react-notifications-component/dist/js/react-notifications.dev.js:21:30)
    at Module../src/react-notification-component.js (/var/www/html/rebion/react/node_modules/react-notifications-component/dist/js/react-notifications.dev.js:2164:81)
    at __webpack_require__ (/var/www/html/rebion/react/node_modules/react-notifications-component/dist/js/react-notifications.dev.js:21:30)
    at /var/www/html/rebion/react/node_modules/react-notifications-component/dist/js/react-notifications.dev.js:85:18
    at Object.<anonymous> (/var/www/html/rebion/react/node_modules/react-notifications-component/dist/js/react-notifications.dev.js:88:10)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)`

**Comment by teodosii** [comment]
Hmm, that is weird. The library code does not need `window` object until `componentDidMount` gets called. Looking at the generated `react-notifications.dev.js` looks like webpack at line 1355 is doing this

```js
return window && document && document.all && !window.atob;
```

I wonder if this is the root cause. Try importing the prod version directly from node-modules, maybe that should help.

```js
import ReactNotifications from react-notifications-component/dist/js/react-notifications.prod.js
```

See if that helps and come with a reply. I know `style-loader` had a similar issue and in that case I'll need to change it to use a different loader to suport SSR.

**Comment by teodosii** [comment]
This might be a temporary fix as `style-loader` is not used in the prod config. Will do a patch for this.

**Comment by teodosii** [comment]
Fixed in `2.1.0` - you can now get back to the initial path to have validators included in development.

**Comment by hannanstd** [comment]
thanks
